dev-libs/hyperscan/hyperscan-5.2.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake python-r1 flag-o-matic
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://www.hyperscan.io/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cpu_flags_x86_ssse3 static-libs"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-util/ragel
+	dev-libs/boost
+	net-libs/libpcap"
+BDEPEND="${RDEPEND}"
```

Comment:
This looks wrong, libcpap and ragel could in theory work like this, but what about boost? Are you sure boost isn't needed in DEPEND?

Reply:
Honestly, I didn't pay much attention to the ebuild because I believed `5.2.0` is ok.

Anyway, `dev-util/ragel` should stay in `BDEPEND` because it is used to generate `.cpp` from `.rl` files. `net-libs/libpcap` is optional dependency only for some example codes, there is no need to compile them, therefore I removed it. `hyperscan` uses header-only parts of the Boost and it should be placed to `DEPEND` but I placed it to the `RDEPEND` as well because I find it right.

Btw, I also create `src_test` function to run hyperscan unit tests.